src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -3874,4 +3920,31 @@ int IDebugSettingsCallback110.ShouldSuppressImplicitToStringCalls(out int pfSupp
             }
         }
     }
+
+    internal class SendInvalidateRequest : DebugRequestWithResponse<SendInvalidateArguments, SendInvalidateResponse>
+    {
+ 
+        public SendInvalidateRequest(): base("sendInvalidate")
+        {
+        }
+    }
+
+    internal class SendInvalidateResponse : ResponseBody
+    {
+        public sealed class Body
+        {
+            public string result;
+        }
+
+        public Body body = new Body();
+    }
+
+    internal class SendInvalidateArguments : DebugRequestArguments
+    {
+
+        public string Areas { get; set; }
```

Comment:
I missed this in an earlier review, are you able to use `InvalidatedAreas`?

This would make lines 143 - 160 be:
```csharp
invalidated.Areas.Add(responder.Arguments.Areas)
```

Reply:
done